app/controllers/app/user/create.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Projects4Me Copyright (c) 2017. Licensing : http://legal.projects4.me/LICENSE.txt. Do not remove this line
+*/
+
+import PrometheusCreateController from "prometheus/controllers/prometheus/create";
+import { tracked } from "@glimmer/tracking";
+import { htmlSafe } from '@ember/template';
+
+/**
+ * The controller for user create page.
+ *
+ * @class AppUserCreateController
+ * @namespace Prometheus.Routes
+ * @module App.User
+ * @extends PrometheusCreateController
+ * @author Rana Nouman <ranamnouman@gmail.com>
+ */
+export default class AppUserCreateController extends PrometheusCreateController {
+
+    /**
+     * This is the module for which we are trying to create.
+     *
+     * @property module
+     * @type String
+     * @for AppUserCreateController
+     * @protected
+     */
+    module = 'user';
+
+    /**
+     * The language selected by user.
+     *
+     * @property selectedLanguage
+     * @for AppUserCreateController
+     * @type String
+     * @protected
+     */
+    @tracked selectedLanguage = null;
+
+    /**
+     * The list of language that will be used to show a dropdown of different languages.
+     *
+     * @property languagesList
+     * @for AppUserCreateController
+     * @type Array
+     * @protected
+     */
+    languagesList = [
```

Comment:
I think that this should be in the language service

Reply:
Switching b/w languages can be concern, which is already resolved by `ember-intl` service. So why to create another service that deals with language?

Reply:
I think so, as we will have to tell ember-intl which language to use. Also application wide we will need to allow people to see which language is supported. We will have to load the relevant language from back-end later on as well. So its better to have a separate service that can, for now, at least maintain the following.

1. Supported Languages
2. Current Language.